osu.Game/Overlays/Rankings/SpotlightSelector.cs

```diff
@@ -36,50 +39,60 @@ public IEnumerable<APISpotlight> Spotlights
             set => dropdown.Items = value;
         }
 
+        protected override bool StartHidden => true;
+
         private readonly InfoColumn startDateColumn;
         private readonly InfoColumn endDateColumn;
+        private readonly InfoColumn mapCountColumn;
+        private readonly InfoColumn participantsColumn;
+        private readonly Container content;
 
         public SpotlightSelector()
         {
             RelativeSizeAxes = Axes.X;
-            Height = 100;
-
-            InternalChildren = new Drawable[]
+            Add(content = new Container
             {
-                background = new Box
-                {
-                    RelativeSizeAxes = Axes.Both,
-                },
-                new Container
+                Height = height,
+                RelativeSizeAxes = Axes.X,
+                Children = new Drawable[]
                 {
-                    RelativeSizeAxes = Axes.Both,
-                    Padding = new MarginPadding { Horizontal = UserProfileOverlay.CONTENT_X_MARGIN, Vertical = 10 },
-                    Children = new Drawable[]
+                    background = new Box
                     {
-                        dropdown = new SpotlightsDropdown
-                        {
-                            Anchor = Anchor.TopCentre,
-                            Origin = Anchor.TopCentre,
-                            RelativeSizeAxes = Axes.X,
-                            Current = Current,
-                            Depth = -float.MaxValue
-                        },
-                        new FillFlowContainer
+                        RelativeSizeAxes = Axes.Both,
+                    },
+                    new Container
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Padding = new MarginPadding { Horizontal = UserProfileOverlay.CONTENT_X_MARGIN, Vertical = 10 },
+                        Children = new Drawable[]
                         {
-                            Anchor = Anchor.BottomRight,
-                            Origin = Anchor.BottomRight,
-                            AutoSizeAxes = Axes.Both,
-                            Direction = FillDirection.Horizontal,
-                            Spacing = new Vector2(15, 0),
-                            Children = new Drawable[]
+                            dropdown = new SpotlightsDropdown
                             {
-                                startDateColumn = new InfoColumn(@"Start Date"),
-                                endDateColumn = new InfoColumn(@"End Date"),
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                RelativeSizeAxes = Axes.X,
+                                Current = Current,
+                                Depth = -float.MaxValue
```

Comment:
I feel like this shouldn't be explicitly set for the dropdown. Dropdowns should by their nature have the dropdown content at the forefront of the screen always (can't think of any reasonable exceptions). Which is not to say this code is *wrong*, I just don't want this popping up with every dropdown usage - that could just mean the current default is not chosen correctly.

Reply:
In most cases dropdown is a part of autosized drawable

Reply:
At this point I don't even remember, why I set it there. Will revisit and reply here

Reply:
I checked - it's because of the dropdown list getting covered by the neighbouring `FillFlowContainer` here when the dropdown is opened. The list should always have min depth when opened IMO.